components/browser/state/src/main/java/mozilla/components/browser/state/state/content/PermissionHighlightsState.kt

```diff
@@ -4,13 +4,17 @@
 
 package mozilla.components.browser.state.state.content
 
+import android.os.Parcelable
+import kotlinx.android.parcel.Parcelize
+
 /**
  * Value type that represents any information about permissions that should
  * be brought to user's attention.
  *
  * @property isAutoPlayBlocking indicates if the autoplay setting
  * disabled some web content from playing.
  */
+@Parcelize
 data class PermissionHighlightsState(
     val isAutoPlayBlocking: Boolean = false
-)
+) : Parcelable
```

Comment:
Does this need to be `Parcelable`? I couldn't see in the PR why we added this.

Reply:
It's just for convenience purpose, consumer can past the class across Activity/fragments without the need of creating  their own to add `Parcelable` as the class is small I think it should be OK, what do you think? :)

Reply:
You can see an usage [here](https://github.com/mozilla-mobile/fenix/pull/17101)